packages/lib/Logger.ts

```diff
@@ -252,13 +252,17 @@ class Logger {
 				// eslint-disable-next-line promise/prefer-await-to-then -- Old code before rule was applied
 				writeToFileMutex_.acquire().then((r: Function) => {
 					release = r;
-					return Logger.fsDriver().appendFile(target.path, `${line.join(': ')}\n`, 'utf8');
+					// eslint-disable-next-line promise/prefer-await-to-then
+					return Logger.fsDriver().appendFile(target.path, `${line.join(': ')}\n`, 'utf8').then(() => {
+						this.cleanLogFile(target.path);
+					});
 					// eslint-disable-next-line promise/prefer-await-to-then -- Old code before rule was applied
 				}).catch((error: any) => {
 					console.error('Cannot write to log file:', error);
 					// eslint-disable-next-line promise/prefer-await-to-then -- Old code before rule was applied
 				}).finally(() => {
 					if (release) release();
+					this.deleteOlderLogFiles(target.path);
```

Comment:
Some logic might be needed here to make sure that multiple copies of `deleteOlderLogFiles` aren't running at once (because `Logger.fsDriver().readDirStats(logFileDir)` isn't synchronous).

Reply:
I think this might be related to your comment above:
> When I rename the log file to log-timestamp.txt sometimes appears an error in the console, because it seems that it runs many times in a row and there isn't a log file to be renamed;